cluster/examples/kubernetes/ozone/install-operator-rbac.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: objectstores.ozone.rook.io
+spec:
+  group: ozone.rook.io
+  names:
+    kind: ObjectStore
```

Comment:
You might consider naming it OzoneObjectStore. If there is some other type called ObjectStore the user could just get unpredictable results with `kubectl` commands. This is why we ended up with most Rook CRD types having the provider prefix such as CephObjectStore. But it's still your choice.

Reply:
I just noticed this was already renamed in the design doc, so I must have made this suggestion in the past already. :)

Reply:
Sure, I think I copied an earlier version of the CRD. Now it's prefixed.